header-rewrite-filter/header_processor.h

```diff
@@ -58,6 +66,27 @@ class SetPathProcessor : public HeaderProcessor {
   std::string request_path_; // path to set
 };
 
+class SetBoolProcessor : public Processor {
+public:
+  virtual ~SetBoolProcessor() {}
+  virtual absl::Status parseOperation(std::vector<absl::string_view>& operation_expression);
+  virtual bool executeOperation() const;
+  const std::string& getBoolName() const { return bool_name_; }
+
+private:
+  void setBoolName(absl::string_view bool_name) { bool_name_ = std::string(bool_name); }
+
+  const std::pair<std::string, std::string>& getStringsToCompare() const { return strings_to_compare_; }
+  void setStringsToCompare(std::pair<absl::string_view, absl::string_view> strings_to_compare);
+
+  Utility::MatchType getMatchType() const { return match_type_; }
+  void setMatchType(Utility::MatchType match_type) { match_type_ = match_type; }
```

Comment:
These getter/setter funcs are all private - is there any reason not to remove these and have callers set the variables directly?

Reply:
Yeah, that's a good point -- I was originally planning on using these getters and setters in the filter code, but I don't need to with the current implementation. I'll go through and clean up the interface in a separate PR.